UDPServer/SiceUDPServer.cs

```diff
@@ -85,5 +82,19 @@ public void Dispose()
         {
             client.Dispose();
         }
+
+        public void Handle(ServerCommandEvent command)
+        {
+            if(command.Command)
+            {
+                ConnectionIP = command.ConnectionIP;
+                ConnectionPort = command.ConnectionPort;
+                this.Connect();
+            }
+            else
+            {
+                this.Disconnect();
+            }
```

Comment:
how bout using a guard condition?

Reply:
Seems a bit overkill and really mess up the logic in this case